### ISSUE TITLE
[Confluence Cloud] Support for Tasks

### ISSUE THREAD

**Issue by philr-biz** [atlassian-api/atlassian-python-api#1065]
As far as I can tell, there is currently no support for retrieving tasks from Confluence cloud.

I'm happy to look into adding this, but wanted to check if this is already planned before I start.

For clarity, I'm talking about this endpoint: https://developer.atlassian.com/cloud/confluence/rest/v1/api-group-inline-tasks/

**Comment by pschmutz** [comment]
@philprobinson84 I also considered adding this. Did you already implement something?

**Comment by philprobinson84** [comment]
> @philprobinson84 I also considered adding this. Did you already implement something?

Nope, I never got around to it. Still interested in having support for this though.

**Comment by gonchik** [comment]
Added that one: 
https://developer.atlassian.com/cloud/confluence/rest/v2/api-group-task/#api-group-task

you can follow there:
https://github.com/atlassian-api/atlassian-python-api/commit/735eb0191e7cab41ee8c20b3d1136d1447ec388d
